java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -1791,6 +1792,18 @@ public T time() {
         return getThis();
     }
 
+    /**
+     * Synchronously saves the DataBase.<br>
+     * This command is not available in transaction starting from Redis version 7.
```

Comment:
Is it ok to leave it here?

Reply:
Is this the correct wording? 
It's no longer available?

Reply:
```
127.0.0.1:6379> multi
OK
127.0.0.1:6379(TX)> save
(error) ERR Command not allowed inside a transaction
```